randovania/games/prime2/json_data/Sanctuary Fortress.txt

```diff
@@ -3429,6 +3439,8 @@ Extra - in_dark_aether: True
                       Any of the following:
                           Dark World Damage ≥ 130
                           Dark World Damage ≥ 60 and Activate Safe Zone
+          # https://www.youtube.com/watch?v=u4gctVgspEc
+          Screw Attack and Space Jump Boots and Slope Jump (Expert) and Dark World Damage ≥ 90 and Use Screw Attack (Space Jump)
```

Comment:
Damage seems a bit high no?

Reply:
I compared it with the BSJ into SA which is >= 90. With activate safe Zone it's 40. I was thinking since it's expert trick you probably need a bit extra health, but if we only consider first try attempts it is probably lower (maybe like >= 60 ?)

Reply:
BSJ needs more because you wait a bit longer. Could lower it to like 75